packages/@webex/plugin-meetings/test/integration/spec/space-meeting.js

```diff
@@ -59,7 +59,7 @@ skipInNode(describe)('plugin-meetings', () => {
         })
     );
 
-    it('Alice starts a space meeting', () =>
+    it('Alice starts a space meeting', async () =>
```

Comment:
Why async when I do not see an await inside?

Reply:
Same this is also not required. removed this. This is flaky thats why I had put the async